### PR TITLE
Update README and DevGuide (value proposition, user stories, NFRs)

### PR DESCRIPTION
**Pull request by dorcastan**
Change format of `README.md` introduction and update miscellaneous Developer Guide appendices.

Addresses #8

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -277,15 +279,22 @@ Refer to the guide <<DevOps#, here>>.
...
 
-* has a need to manage a significant number of bookmarks
-* prefer desktop apps over other types
-* can type fast
-* prefers typing over mouse input
+* needs to manage a significant number of bookmarks
+* needs to make notes on webpage content quite frequently
+* needs reminders to submit forms or visit websites
+* prefers desktop apps over other types
+* can type fast and prefers typing over mouse input
```
Comment (openorclose) [inline docs/DeveloperGuide.adoc]:
```suggestion
* types fast and prefers typing over mouse input
```

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -277,15 +279,22 @@ Refer to the guide <<DevOps#, here>>.
...
 * is reasonably comfortable using CLI apps
```
Comment (openorclose) [inline docs/DeveloperGuide.adoc]:
```suggestion
* likes using CLI apps
```

maybe? trying to do the parallel thing

File: docs/DeveloperGuide.adoc [hunk 3]
```diff
@@ -277,15 +279,22 @@ Refer to the guide <<DevOps#, here>>.
...
 
-*Value proposition*: manage bookmarks faster than a typical mouse/GUI driven app
+*Value proposition*:
```
Comment (openorclose) [inline docs/DeveloperGuide.adoc]: Should we add 'our' here to match with 'our target user'?

Comment (dorcastan) [reply docs/DeveloperGuide.adoc]:
```suggestion
*Our value proposition*:
```

**Pull request merged**